stable/xray/values.yaml

```diff
@@ -367,6 +367,8 @@ common:
   # xrayVersion:
   xrayUserId: 1035
   xrayGroupId: 1035
+  routerUserId: 1117
```

Comment:
I think this belongs in the router section of the values.yaml, i.e. .Values.router.userId and .Values.router.groupId
The common section is intended for values that are common for all of xray’s micro services